src/i18n/en-US.json

```diff
@@ -959,11 +959,14 @@
   "modalUserBlockHeadline": "Block {{user}}?",
   "modalUserBlockMessage": "{{user}} won’t be able to contact you or add you to group conversations.",
   "modalUserBlockedForLegalHold": "This user is blocked due to legal hold. [link]Learn more[/link]",
+  "modalUserCannotAcceptConnectionMessage": "Connection request could not be accepted",
   "modalUserCannotBeAddedHeadline": "Guests cannot be added",
-  "modalUserCannotConnectLegalHoldHeadline": "Cannot connect to this user",
```

Comment:
The localization team requested to change this title for the no-longer-federating case.

We are using the new copy "Connecting not possible" as a title for every case